python/sglang/srt/layers/moe/fused_moe_triton/triton_kernels_moe.py

```diff
@@ -169,11 +169,3 @@ def triton_kernel_moe_forward_fake(
     block_shape: Optional[list[int]] = None,
 ) -> torch.Tensor:
     return torch.empty_like(hidden_states)
-
-
-direct_register_custom_op(
```

Comment:
What is the error? If we do not have this op registration, do we still need the fake function?

Reply:
```
  File "/host_home/primary_synced/sglang/python/sglang/srt/layers/quantization/unquant.py", line 137, in __init__
    from sglang.srt.layers.moe.fused_moe_triton.triton_kernels_moe import (
  File "/host_home/primary_synced/sglang/python/sglang/srt/layers/moe/fused_moe_triton/triton_kernels_moe.py", line 204, in <module>
    direct_register_custom_op(
  File "/host_home/primary_synced/sglang/python/sglang/srt/utils.py", line 1715, in direct_register_custom_op
    schema_str = torch.library.infer_schema(op_func, mutates_args=mutates_args)
  File "/usr/local/lib/python3.10/dist-packages/torch/_library/infer_schema.py", line 127, in infer_schema
    if annotation_type.__origin__ is tuple:
AttributeError: type object 'TopKOutput' has no attribute '__origin__'

Reply:
I think we should keep it. How about registering `triton_kernel_fused_experts` instead?

Reply:
I do not find where this is used